test/unittest/unittest_nntrainer_lazy_tensor.cpp

```diff
@@ -0,0 +1,122 @@
+/* SPDX-License-Identifier: Apache-2.0-only
+ * Copyright (C) 2020 Jihoon Lee <jihoon.it.lee@samsung.com>
+ *
+ * @file	unittest_nntrainer_lazy_tensor.cpp
+ * @date	05 Jun 2020
+ * @brief	A unittest for nntrainer_lazy_tensor
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Jihoon Lee <jihoon.it.lee@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ */
+
+#include "nntrainer_test_util.h"
+#include "util_func.h"
+#include <lazy_tensor.h>
+#include <fstream>
+#include <nntrainer_error.h>
+#include <tensor.h>
+#include <tensor_dim.h>
+
+TEST(nntrainer_LazyTensor, LazyTensor_01_p) {
+  int batch = 3;
+  int height = 3;
+  int width = 10;
+  int channel = 1;
+
+  nntrainer::Tensor target(batch, channel, height, width);
+  GEN_TEST_INPUT(target, i * (batch * height) + j * (width) + k + 1 + l);
+
+  nntrainer::LazyTensor defed(target);
```

Comment:
what's meaning of `defed`?

Reply:
It meant `deferred`. I think the name is unclear, I changed it to `delayed`. Thanks